modules/ghostty/hm.nix

```diff
@@ -0,0 +1,55 @@
+# Documentation is available at:
+# - https://ghostty.org/docs/config/reference
+# - `man 5 ghostty`
+{ config, lib, ... }:
+{
+  options.stylix.targets.ghostty.enable = config.lib.stylix.mkEnableTarget "Ghostty" true;
+
+  config = lib.mkIf (config.stylix.enable && config.stylix.targets.ghostty.enable) {
+    programs.ghostty = {
+      settings =
+        let
+          inherit (config.stylix) fonts opacity;
+        in
+        {
+          theme = "stylix";
+          font-family = [
+            fonts.monospace.name
+            fonts.emoji.name
+          ];
+          font-size = fonts.sizes.terminal;
+          background-opacity = opacity.terminal;
+        };
+      themes."stylix" =
```

Comment:
```suggestion
      themes.stylix =
```

Reply:
Forgot to remove the quotes while experimenting with something.